Rakefile

```diff
@@ -14,3 +12,15 @@ task :test_isolated do
     abort unless $?.success?
   end
 end
+
+begin
+  require "rubocop/rake_task"
+
+  RuboCop::RakeTask.new(:rubocop) do |task|
+    task.options = ['--display-cop-names']
+  end
+rescue LoadError
```

Comment:
I think rubocop should be separated as a single job, makes no sense to test it for all rubies.